### PR TITLE
2017 07 pv improve activity stream timestamps

### PR DESCRIPTION
**Pull request by vellip**

### CODE REVIEW

File: meinberlin/assets/scss/components/_action.scss [hunk 1]
```diff
@@ -7,10 +7,27 @@
...
+    border: 1px solid $border-color;
+
+    @media (min-width: $breakpoint) {
+        padding: 1em;
+    }
+
+    h2 {
+        margin-top: 0;
+    }
+}
```
Comment (xi) [inline meinberlin/assets/scss/components/_action.scss]: I don't like having two components on a single DOM element. Can we change this to `.block--bordered`?

File: meinberlin/assets/scss/components/_action.scss [hunk 2]
```diff
@@ -7,10 +7,27 @@
     font-size: $font-size * 100%;
 }
 
+.actions {
+    padding: 0;
```
Comment (xi) [inline meinberlin/assets/scss/components/_action.scss]: Having no padding looks strange IMHO.

File: apps/actions/assets/timestamps.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+$(function () {
+  const $times = $('time.relative')
+
+  $times.each((i, e) => {
+    let datetime = moment($(e).attr('datetime'))
+
+    if (datetime.isAfter(moment().subtract(7, 'days'))) {
+      e.textContent = datetime.fromNow()
+    }
```
Comment (xi) [inline apps/actions/assets/timestamps.js]: Nice solution!

File: meinberlin/assets/scss/components/_action.scss (hunk 2 above)
Comment (vellip) [inline meinberlin/assets/scss/components/_action.scss]: how so?

File: meinberlin/assets/scss/components/_action.scss (hunk 1 above)
Comment (vellip) [inline meinberlin/assets/scss/components/_action.scss]: What two components?

Comment (xi) [inline meinberlin/assets/scss/components/_action.scss]: `block l-center-6 actions` -> `block` and `actions`

File: meinberlin/assets/scss/components/_action.scss (hunk 2 above)
Comment (vellip) [inline meinberlin/assets/scss/components/_action.scss]: You were right, don't know what I was thinking there.

**Pull request merged**